schemas/aws/account-1.yml

```diff
@@ -2,90 +2,166 @@
     type: string
+    description: |
+      The unique name of the AWS account. Used to identify the account
+      across the system.
+
   description:
     type: string
+    description: |
+      A detailed description of the AWS account, including its intended
+      use, environment, or any relevant context.
+
   consoleUrl:
     type: string
     format: uri
+    description: |
+      The URL to access the AWS console for this account.
+
   uid:
     type: string
+    description: |
+      The unique identifier (UID) for the AWS account.
+
   resourcesDefaultRegion:
     type: string
+    description: |
+      The default AWS region for resources created in this account.
+
   supportedDeploymentRegions:
     type: array
     items:
       type: string
-      description: "The regions where the account is enabled and can be used."
+      description: |
+        The regions where the account is enabled and can be used for
+        deployments.
+    description: |
+      A list of AWS regions where deployments are supported for this account.
+
   providerVersion:
     type: string
+    description: |
+      The version of the AWS provider used for managing this account.
+
   terraformUsername:
     type: string
+    description: |
+      The username used by Terraform to manage resources in this account.
+
   accountOwners:
     type: array
+    description: |
+      A list of individuals or teams responsible for this AWS account.
+      Each entry must include a name and email address for contact.
     items:
       type: object
       properties:
         name:
           type: string
+          description: |
+            The name of the account owner.
         email:
           type: string
           format: email
+          description: |
+            The email address of the account owner.
       required:
       - name
       - email
+
   securityContact:
     type: object
+    description: |
+      Contact information for the security point of contact for this account.
+      Used for incident response and compliance notifications.
     properties:
       name:
         type: string
-        description: "The name of the security contact."
+        description: |
+          The name of the security contact.
       title:
         type: string
-        description: "The title of the security contact. Default use the name."
+        description: |
+          The title of the security contact. Defaults to the name if not set.
       email:
         type: string
         format: email
+        description: |
+          The email address of the security contact.
       phoneNumber:
         "$ref": "/common-1.json#/definitions/phoneNumber"
+        description: |
+          The phone number of the security contact.
     required:
     - name
     - email
     - phoneNumber
+
   automationToken:
     "$ref": "/common-1.json#/definitions/vaultSecret"
+    description: |
+      Reference to the Vault secret containing the automation token for
```

Comment:
```suggestion
      Reference to the Vault secret containing the credentials for
```

Reply:
I admit I'm confused with these suggestions. Why add descriptions with details that do not match the property name?